source/beamline/BioXAS/BioXASM1MirrorMaskState.cpp

```diff
@@ -11,8 +11,8 @@ BioXASM1MirrorMaskState::BioXASM1MirrorMaskState(const QString &name, QObject *p
 
 	// Setup basic value options.
 
-	addValueOption(Open, "Open", 0, 0.01, 10); // these are 'pretend' values, for now. should be safe to use them, as this control does not support moving.
-	addValueOption(Closed, "Closed", 0, 0, 0.01);
+	addValueOption(Closed, "Closed", 0.0, 0.0, 0.01);
+	addValueOption(Open, "Open", 0.01, 10.0);
```

Comment:
With this call, the default optionSetpoint for the `Open` option is "-1". But the original codes the value of optionSetpoint is "0". Is this "-1" correct?